lightspark-remote-signing/src/response.rs

```diff
@@ -228,7 +228,7 @@ impl Response {
     pub fn set_invoice_payment_hash_response(
         invoice_id: &str,
         payment_hash: &str,
-        pre_image_nonce: &str,
+        pre_image_nonce: Option<&str>,
```

Comment:
Any implications about this being backwards-incompatible (for uniffi, etc)?

Reply:
This should be fine, as long as I update the uniffi with this.